marimo/_plugins/stateless/image.py

```diff
@@ -158,4 +161,18 @@ def image(
         }
     )
     img = h.img(src=resolved_src, alt=alt, style=styles)
+    if caption:
```

Comment:
This should be

```suggestion
    if caption is not None:
```

based on https://google.github.io/styleguide/pyguide.html#2144-decision